doc/migrate-to-pymongo4.rst

```diff
@@ -174,6 +174,14 @@ can be changed to this::
 
     names = client.list_database_names()
 
+``directConnection`` defaults to False
+..........................................
+- ``directConnection`` defaults to False instead of None, allowing for the
```

Comment:
Same comment as the changelog.rst except that this file doesn't need the `-` at the beginning of the line.

Reply:
Done.